packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,76 @@
+:local {
+  .field {
+    border: 0;
+    font-family: var(--terra-form-field-font-family, inherit);
```

Comment:
I think we can remove this font-family property here too, unless UX wants the ability to make field a different font than what we set in [base](https://github.com/cerner/terra-core/pull/988/files#diff-cf7e9b185f60dd895f755538bcd63363).

Reply:
Yes the theme font-family for Field are different for both clinical & consumer